### PR TITLE
Broken homepages

### PR DESCRIPTION
**Pull request by Moisan**
This PR solves all broken homepage stated in issue #41610 except `minbif`, `par2tbb` and `trafshow`.

### CODE REVIEW

File: Library/Formula/freealut.rb [hunk 1]
```diff
@@ -36,3 +36,4 @@ def install
  
  ################################################################################
  # Checks for header files.
+
```
Comment (DomT4) [inline Library/Formula/freealut.rb]: Careful with the whitespacing here. Inline patches are very delicate about accidental whitespacing.

Comment (Moisan) [inline Library/Formula/freealut.rb]: Sorry, my editor wasn't correctly configured for whitespaces, everything should be fine now.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by DomT4** [comment]
The merge didn't have a `Closes #45798` line in the final commit for some reason, so closing here manually.

**Pull request closed without merging**